cmd/build/v2/build.go

```diff
@@ -354,9 +366,17 @@ func (bc *OktetoBuilder) addVolumeMounts(ctx context.Context, manifest *model.Ma
 
 	buildInfoCopy := manifest.Build[svcName].Copy()
 	buildInfoCopy.Image = ""
-	buildHash := bc.hasher.hashService(buildInfoCopy)
 
-	tagToBuild := newImageWithVolumesTagger(bc.Config).getServiceImageReference(manifest.Name, svcName, buildInfoCopy, buildHash)
+	var err error
+	var buildHash string
```

Comment:
same, when smart build is not enabled this value is empty. Should we use another approach to get the `buildhash` ?